### DISCUSSION TITLE
Introductions :name_badge:

### DISCUSSION THREAD

**Discussion by carletex** [BuidlGuidl/batch0.buidlguidl.com#1] [General]
Hi all!

My name is Carlos, I've been part of the BuidlGuidl since 2021. I've been a software developer for more than 10 years (mainly PHP, JS, CSS) and went _onchain_ a couple of years ago.

During this time I've built things like the [SpeedRunEthereum](https://speedrunethereum.com/) platform, the [BuidlGuidl App](https://app.buidlguidl.com/), [ABI Ninja](https://abi.ninja/) & [Scaffold-ETH 2](https://github.com/scaffold-eth/scaffold-eth-2). Inside the BuidlGuild, I also run the [Sand Garden](https://sandgarden.buidlguidl.com/).

I love learning and building things 👾

Alongside other BG members, I'll be helping you all to go through this batch. Hope you all have fun!

**Comment by edakturk14** [comment]
Hey everyone! It's Eda here, I've been part of the BuidlGuidl for almost 1.5 years! I'm a developer and I spend a lot of time [writing](https://mirror.xyz/edatweets.eth) about different topics and concepts in crypto. I enjoy working on developer education and creating content/documentation.

I'm very excited for this batch, hope that it will be a very fun project. If you have any questions, feel free to ask anytime!

**Comment by ZakGriffith** [comment]
Hey all, I'm Zak!  I've been with the BuidlGuidl for a couple years now.  I started out assisting with SpeedRunEthereum, but now I also help out in many other projects and places.  I'll be assisting this batch with along with the other mentors.  Let's make this into a good learning environment and have you come out on the other side knowing more than before!

**Comment by YanVictorSN** [comment]
Hi everyone! I'm Yan. I'm from Brazil and new to on-chain development. I'm very excited to learn and contribute in this journey. And in the future, bring this environment to more Brazilians.

**Comment by solo1g** [comment]
Hey all,
I'm Shreyansh, I've started with SpeedRunEthereum challenges just a few days back so super new to the space but it's been fun so far. Hope I get to learn more though this.

**Comment by metoinside** [comment]
Hello folks,  
  
Metin is here from Turkiye. I have been working on various projects as a freelance technical product manager on Upwork during the day but blockchain-focused developer on the night & weekends. Since the beginning of 2023, I've been a big fan of the ScaffoldETH and trying to learn & contribute in some ways.

**Reply by drained69** [reply]
Hi everyone, 

I'm Akwuba Chris from Nigeria. My goal here is to equip myself with better knowledge which would make me useful in the web3 space, contribute with my peers here as I believe collaboration is very vital to growth, and also make amazing connections here.

**Comment by positonic** [comment]
Hey Everyone!

I'm a builder and product leader 🛠 who's passionate about using technology to improve co-ordination in order to fund the commons, pay for public goods and address the greater issues in our time including - social and environmental problems and funding open source software.

I believe public blockchains and Zero Knowledge enable co-ordination and collaboration in ground breaking ways, and I'm always looking to unlock new portals that didn't previously exist in order to beam us into a better future.

Previous work includes:

🌱 Co-founder of Toucan.earth, with aims to rapidly scale the impact of planet-positive projects by turning carbon credits into tokens on public blockchains

🌴 Founding member of KlimaDAO.finance (🌳,🌳), which fights climate change through it's digital carbon backed currency

❤️ Tech lead at Giveth.io

⚗️ Tech lead on Copper at Alchemist

Let's build out own utopian future where technology serves as a catalyst for positive change for us all! 🌿🌐

[This is my Twitter](https://twitter.com/positonic)

I'm currently pretty excited about Allo protocol, Impact evaluation, ZK, and TokenBound accounts.... oh and I'm obviously a little ADHD 😆 .... oh and I'm Irish.... we like self depreciating humour 😉

**Comment by miralsuthar** [comment]
Hello everyone!

I'm Miral from India. I have been the frontend developer for couple of years and now I want to explore blockchain and smart contract development. I'm very excited to learn and contribute in this journey.

**Comment by andrewdavidkay** [comment]
Hi everyone! 👋

My name is Andy, or on crypto twitter people call me swol (short for my username swolidity and i'm big into fitness). I have a programming background but want to dive deeper into web3 development. I'm excited and grateful to be here with all of you!
